index.js

```diff
@@ -474,4 +475,32 @@ var ab2hex = function (ab) {
   return new Buffer(buffer).toString('hex')
 }
 
+// wrap around webcrypto.getRandomValues() to use high-quality PRNG, when available
+var getRandomValues = function (ab) {
+  var err, i, j, octets
+
+  if (!ab.BYTES_PER_ELEMENT) {
+    err = new Error()
+    err.name = 'TypeMisMatchError'
+    throw err
+  }
+  if (ab.length > 65536) {
+    err = new Error()
+    err.name = 'QuotaExceededError'
+    throw err
+  }
+
+  octets = crypto.randomBytes(ab.length * ab.BYTES_PER_ELEMENT)
+
+  if (ab.BYTES_PER_ELEMENT === 1) ab.set(octets)
+  else {
```

Comment:
i disagree. i want to avoid a situation where someone grabs the code and get gets burnt. with the exception of an if-conditional test, there is no performance hit...